typing/typetexp.ml

```diff
@@ -309,11 +409,7 @@ and transl_type_aux env policy styp =
   | Ptyp_alias(st, alias) ->
       let cty =
         try
-          let t =
-            try List.assoc alias !univars
-            with Not_found ->
-              instance (fst(TyVarMap.find alias !used_variables))
-          in
+          let t = TyVarEnv.lookup_local alias in
```

Comment:
`lookup_local` is defined as taking an `instance` of the univar, this is not what was going on in the success case here. Is this a change of behavior?

Reply:
Good catch! Actually, some further analysis reveals we never need `instance` here -- it was redundant before my patch. I will add a commit shortly that makes this more apparent and removes the call.